client/reader/tag-stream/controller.js

```diff
@@ -16,16 +16,31 @@ const analyticsPageTitle = 'Reader';
 
 export default {
 	tagListing( context ) {
-		var TagStream = require( 'reader/tag-stream/main' ),
-			basePath = '/tag/:slug',
-			fullAnalyticsPageTitle = analyticsPageTitle + ' > Tag > ' + context.params.tag,
-			tagSlug = trim( context.params.tag )
-				.toLowerCase()
-				.replace( /\s+/g, '-' )
-				.replace( /-{2,}/g, '-' ),
-			encodedTag = encodeURIComponent( tagSlug ).toLowerCase(),
-			tagStore = feedStreamFactory( 'tag:' + tagSlug ),
-			mcKey = 'topic';
+		var config = require( 'config' );
+
+		if ( config.isEnabled( 'reader/topics' ) ) {
```

Comment:
why the if / else? the only thing different is the base path, ya? and I assume the /tag links work fine even with `reader/topics` enabled?